platform/security/rest/security-rest-cxfwrapper/src/test/java/org/codice/ddf/cxf/SecureCxfClientFactoryTest.java

```diff
@@ -181,6 +186,27 @@ public void validateConduit() throws SecurityServiceException {
                 .isDisableCNCheck(), is(true));
     }
 
+    @Test
+    public void testHttpsClientWithSystemProperty() {
+        System.setProperty("property", TEST_SYSTEM_PROPERTY);
+        // positive case
+        SecureCxfClientFactory<IDummy> secureCxfClientFactory = new SecureCxfClientFactory<>(
+                SECURE_ENDPOINT_SYSTEM_PROPERTIES,
+                IDummy.class);
+        Client unsecuredClient = WebClient.client(secureCxfClientFactory.getClient());
+        assertTrue(unsecuredClient.getBaseURI()
+                .toASCIIString()
+                .equals(SECURE_ENDPOINT));
+        // negative cases
+        IDummy result;
+        result = secureCxfClientFactory.getClientForSubject(getSubject());
+        assertNotNull(result);
```

Comment:
We should be using assertThat and Hamcrest : `assertThat(result, notNullValue())`